docs/src/components/layouts/utility-page-layout/utility-class-tables/index.js

```diff
@@ -1,13 +1,19 @@
 export * as BoxModelDemonstration from './box-model-demonstation';
 export * as ClassValueDemonstration from './class-value-demonstration';
 export * as ColorDemonstration from './color-demonstration';
+export * as DisplayDemonstration from './display-demonstration';
```

Comment:
Why did this need to change? I am confused as these all should be working.

Reply:
they were just orded/alphabetized